account-kit/smart-contracts/src/ma-v2/modules/time-range-module/module.ts

```diff
@@ -0,0 +1,82 @@
+import { encodeAbiParameters, type Address, type Hex } from "viem";
+
+import { timeRangeModuleAbi } from "./abis/timeRangeModuleAbi.js";
+
+const addresses = {
+  default: "0x00", // TO DO: replace with deployed module address?
```

Comment:
As mentioned in other PR, use `zeroAddress` from Viem.

Reply:
`addresses` is no longer part of the module definition, therefore resolving this comment!